pkg/agent/openflow/client.go

```diff
@@ -279,6 +280,19 @@ type Client interface {
 		dstIP net.IP,
 		outPort uint32,
 		igmp ofutil.Message) error
+
+	// InstallTrafficControlMarkFlows installs the flows to mark the packets for a traffic control rule with the traffic control
+	// mark value.
+	InstallTrafficControlMarkFlows(name string, mark uint32, srcOfPorts []uint32, direction v1alpha2.Direction) error
+
+	// UninstallTrafficControlMarkFlows removes the flows for a traffic control rule.
+	UninstallTrafficControlMarkFlows(name string) error
+
+	// InstallTrafficControlInitFlows installs the initial flows for the traffic control mark value.
+	InstallTrafficControlInitFlows(mark uint32, action v1alpha2.Action, dstOfInPort, dstOfOutPort uint32) error
+
+	// UninstallTrafficControlInitFlows removes the flows for the traffic control mark value.
+	UninstallTrafficControlInitFlows(mark uint8) error
```

Comment:
The arg type of mark in `InstallTrafficControlInitFlows` is uint32. `UninstallTrafficControlInitFlows` mark type is uint8?

Reply:
It should be uint32.